### PR TITLE
diff-match-patch

### PR DESCRIPTION
**Pull request by ocefpaf**
See https://pypi.python.org/pypi/diff-match-patch/

### CODE REVIEW

File: recipes/diff-match-patch/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+test:
+  imports:
+    - diff_match_patch
+
+about:
+  home: http://code.google.com/p/google-diff-match-patch/
+  license: Apache 2.0
+  license_file: LICENCE
+  summary: 'The Diff Match and Patch libraries offer robust algorithms to perform the operations required for synchronizing plain text.'
```
Comment (scopatz) [inline recipes/diff-match-patch/meta.yaml]: Long line

Comment (scopatz) [inline recipes/diff-match-patch/meta.yaml]: Well you can use a pipe and then enter a literal block. I am not suggesting changing the text, only that this goes over the line character limit. I guess it isn't a huge deal...

Comment (ccordoba12) [inline recipes/diff-match-patch/meta.yaml]: For Navigator stuff, @goanpeca is the expert, not me :-)

Comment (ocefpaf) [inline recipes/diff-match-patch/meta.yaml]: Good to know. Thanks!

**Pull request merged**